.gitignore

```diff
@@ -6,7 +6,17 @@ Emakefile
 .DS_Store
 _build
 erl_crash.dump
-/doc/README.md
-/doc/erlang.png
-/doc/edoc-info
-/doc/stylesheet.css
+
+# edoc
+doc/README.md
```

Comment:
doc/README.md is probably from edown, not from edoc itself.

Reply:
Ah, ok, added edown